src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/ToolTipProvider.cs

```diff
@@ -89,6 +90,11 @@ private void ProcessDescription(Task<CompletionDescription> obj)
 
                 var description = obj.Result;
                 this.Content = GetTextBlock(description.TaggedParts, _toolTipProvider._typeMap);
+
+                // As of https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/67974?_a=overview 
```

Comment:
I personally don't think you need to cite the PR where the editor started doing this. Just explaining that they do it should be sufficient.